src/stores/HipsQueryStore/HipsQueryStore.ts

```diff
@@ -51,6 +51,16 @@ export enum HipsProjection {
     XPH = "XPH"
 }
 
+export enum HipsConstraint {
```

Comment:
I think it is uncommon to assign the same values in an enum which could cause problems when comparing values. We could use an object instead since it's unnecessary to use an enum here.